modules/qna/src/views/full/Components/ContextSelector.tsx

```diff
@@ -17,7 +17,7 @@ const ContextSelector: FC<Props> = props => {
   const contexts = props.contexts || []
 
   useEffect(() => {
-    props.bp.axios.get('/nlu/contexts').then(({ data }) => setContexts(data))
+    props.bp.axios.get('/nlu/contexts').then(({ data }) => setContexts(data.sort()))
```

Comment:
Suggestion:
You could declare a reusable function, e.g.

```
const updateContexts = contexts => setContexts(contexts.sort())
```
and use that function in the 2 places where you currently call `.sort()`

Reply:
I agree that it's cleaner that way and also makes sure that the local contexts are always sorted! Thanks for the suggestion!